traitsui/editors/api.py

```diff
@@ -7,11 +7,15 @@
 try:
     from .array_editor import ArrayEditor
 except ImportError as e:
-    if 'numpy' not in e.message:
```

Comment:
I would be happier with a simple approach that does not require two `try...except`